server/migrations/20220722184916-remove-event-updatedat.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  async up (queryInterface, Sequelize) {
+    await queryInterface.removeColumn("events", "updatedAt");
+  },
+  async down (queryInterface, Sequelize) {
+    await queryInterface.addColumn("events", "updatedAt", {
+      type: Sequelize.DATE,
+      allowNull: false,
+      defaultValue: new Date(),
```

Comment:
```suggestion
      defaultValue: Sequelize.fn('NOW'),
```

I believe this would be more semantically correct, of course there are no plans to reverse this migration

Reply:
Fixed! Thanks for the tip :pray: